g11n-ws/vip-manager-l10n-agent/src/main/resources/application.properties

```diff
@@ -24,6 +24,8 @@ source.sync.s3.startTime=2020-01-01 13:45:11
 remote.source.schedule.cron=0 0/5 * * * ?
 #csp l10 site addess
 remote.l10n.base.url=https://localhost:8088
+#define the user-agent name, default is NoAgent
+remote.l10n.base.user-agent=NoAgent
```

Comment:
it should have an initial value(e.g sgtn_l10n_agent), having a harcoded 'NoAgent' seems meaningless.
so if there's no 'remote.l10n.base.user-agent' or it's empty, it doesn't need to add the header.

Reply:
done